suite-native/module-send/src/sendOutputsFormSchema.ts

```diff
@@ -194,10 +194,36 @@ export const sendOutputsFormValidationSchema = yup.object({
             }),
         )
         .required(),
+    enableDestinationTagValidation: yup.boolean(),
```

Comment:
nit:  what about this name?
```suggestion
    isRippleDestinationTagEnabled: yup.boolean(),
```

Reply:
d44a49161a

Reply:
d44a49161a